include/MixerLine.h

```diff
@@ -113,4 +115,6 @@ private slots:
 };
 
 
+} // namespace lmms::gui
+
 #endif // MIXERLINE_H
```

Comment:
Can you please change the to `#ifndef` (and the subseq. line) to `MIXERLINE_H`?

Reply:
Done.